packages/playwright/tests/axe-playwright.spec.ts

```diff
@@ -17,16 +17,20 @@ describe('@axe-core/playwright', () => {
   const axeSource = fs.readFileSync(axePath, 'utf8');
   let browser: playwright.ChromiumBrowser;
   const axeTestFixtures = path.resolve(__dirname, 'fixtures');
+  const externalPath = path.resolve(axeTestFixtures, 'external');
   const axeLegacySource = fs.readFileSync(
-    path.resolve(axeTestFixtures, 'external', 'axe-core@legacy.js'),
+    `${externalPath}/axe-core@legacy.js`,
```

Comment:
This path structure could fail on windows

```suggestion
    path.join(externalPath, 'axe-core@legacy.js'),
```